rustfmt-core/tests/target/issue-2445.rs

```diff
@@ -0,0 +1,12 @@
+test!(RunPassPretty {
+    // comment
+    path: "src/test/run-pass/pretty",
+    mode: "pretty",
+    suite: "run-pass",
+    default: false,
+    host: true // comment
+});
+
+test!(Test {
+    field: i32 // comment
```

Comment:
This test seems wrong: we need to preserve a trailing comma if the macro originally has it.